js/modules/k6/html/gen/gen_elements.go

```diff
@@ -6,10 +6,21 @@ import (
 	"go/format"
 	"go/parser"
 	"go/token"
-	"log"
 	"os"
 	"strings"
 	"text/template"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+const (
+	stringTemplate   = "string"
+	urlTemplate      = "url"
+	enumTemplate     = "enum"
+	boolTemplate     = "bool"
+	gojaEnumTemplate = "enum-goja"
```

Comment:
What does this mean?

Reply:
Basically it's then enum template except instead of returning a string and using the first item in the enum/list of templateargs as default, it returns a goja.Value and uses goja.Undefined as the default.

I've renamed it to nullableEnumTemplate and added comments to explain it.